src/infrastructure/jira/inbound-auth/jira-server-symmetric-jwt-token-verifier.ts

```diff
@@ -56,48 +55,38 @@ export class JiraServerSymmetricJwtTokenVerifier {
 	): Promise<{
 		connectInstallation: ConnectInstallation;
 	}> => {
-		try {
-			const tokenSigningAlgorithm = getAlgorithm(token) as unknown;
+		const tokenSigningAlgorithm = getAlgorithm(token) as unknown;
```

Comment:
**note:** While diff looks a quite significant, there is no logical change in this file. The PR:

- Removes `try/catch` to let errors propagate further. 
- Throws `Error` instead of a custom error. *All errors* thrown from this module are mapped to `UnauthorizedResponseStatusError` in the corresponding middleware to avoid exposing auth-related details to a user. If we need at some point a more granular error handling, we can introduce additional errors specific to this module.

Reply:
Thank you for this comment. It made reviewing this file easier 🙂